### DISCUSSION TITLE
Pls help!

### DISCUSSION THREAD

**Discussion by jeff9060** [aclap-dev/video-downloadhelper#391] [Help and Q&A]
### Your question

Exit code: 1

moz-extension://074d3b96-a34e-4809-b8bb-4bf0f886f557/background/main.js:1
receive@moz-extension://074d3b96-a34e-4809-b8bb-4bf0f886f557/background/main.js:1:13157
Zp</callCatchAppNotFound/</<@moz-extension://074d3b96-a34e-4809-b8bb-4bf0f886f557/background/main.js:1:42626

i don't understand still got this error!and don't know how to fix.

### Addon & CoApp version and details

Video DownloadHelper
版本 8.2.1.1 (stable)
瀏覽器地區: zh-TW
穩定版
編譯時間: 2024-02-19
編譯選項: browser=firefox, linuxlic=false, noyt=false, target=mozilla

平台 Win x86-32
瀏覽器 Mozilla Firefox 124.0.2

夥伴軟體版本: VdhCoApp 2.0.19
夥伴軟體執行檔: C:\Program Files\DownloadHelper CoApp\vdhcoapp.exe

**Comment by Mattmjs** [comment]
• Update the extension or try the beta extension
• verify anti virus & firewall are not blocking the companion app
• share a link where the problem happens if it's public, if private post the hit details information.

**Reply by jeff9060** [reply]
I off anti virus & firewall system.And use bate extension.Sometime can download.But download 1 file finish other file will be error.

**Comment by jeff9060** [comment]
already off windows defender firewall and anti virus.
some time can download.but choose many files.finish 1 file other will stop download and error.

**Reply by Mattmjs** [reply]
And what about the other things I suggested ?

**Reply by jeff9060** [reply]
Update the extension or try the beta extension.Already update.
share a link where the problem happens if it's public, if private post the hit details information.Don't know how to do it.

**Reply by jeff9060** [reply]
Video DownloadHelper 8.2.2.8a1 (beta)

**Reply by Mattmjs** [reply]
For a video that is not working, look to the right of the menu
![001](https://github.com/aclap-dev/video-downloadhelper/assets/160297904/cf923817-ce8e-4124-a7e5-a0d8e94c6885)

Then look for details
![details](https://github.com/aclap-dev/video-downloadhelper/assets/160297904/33428c90-8ada-40ce-ac78-1e8cab0edf4f)

**Reply by Mattmjs** [reply]
@jeff9060 , that's not enough. You have to copy everything on that page, use  **Ctrl + A**

**Reply by jeff9060** [reply]
![IMG_20240412_231058](https://github.com/aclap-dev/video-downloadhelper/assets/166532871/f3d72804-0b73-4b36-a090-e1e0d8bf7561)

**Reply by Mattmjs** [reply]
Oh it seems you did everything correct but Github is not behaving. Ok can you just copy the Master Manifest & media Manifest 
links and paste them here.

**Reply by jeff9060** [reply]
https://surrit.com/572fca42-70e3-4298-932b-7e13a61e7217/playlist.m3u8
https://thisav.com/nsfs-244

**Reply by GrampaWildWilly** [reply]
I popped up the context menu on the manifest URL (your first one) & opened it in a new Firefox window.  Firefox offered to download the manifest.  I dismissed the prompt.  Then I opened the VDH menu on the empty Firefox window.  There were 3 HLS variants on offer.  I told VDH to download the 1280x720 variant, the highest resolution of the 3.  That completed successfully in a bit over 1 minute.  It played fine, although I just sampled the video.  It did not look particularly interesting.  Warning: This is a porno, & by all appearances, a rather rough, non-consensual one at that.  Like I say, I just skimmed through it & it seemed rather unappealing.  In any case, VDH had no problem downloading it.  I am running Windows 11 Pro 64-bit version 23H2 build 22631.3447, Firefox 124.0.2 64-bit, licensed VDH 8.2.2.8a1 beta, CoApp 2.0.19.  I do not work for VDH.  I'm just another user.

**Reply by GrampaWildWilly** [reply]
Here's a belated comment about copying VDH's Hit Details into a discussion page.  If you go from the Hit Details browser page where you do a Select All/Copy Text directly to a GitHub discussion browser page where you do a Paste Text, you get something totally useless.  It's not GitHub that's the problem.  It's that the browser is being entirely smarter than we want it to be.  You have to make one step of indirection to make things work the way we expect.  The steps are these:

1. Select All/Copy Text on the Hit Details browser page.
2. Switch tasks to a text editor (Notepad, Notepad++, whatever you use, just NOT a word processor).
3. Paste text.
4. Select All/Copy Text in the text processor.  No, this is not a duplicate step.
5. Switch tasks to the GitHub browser page.
6. Paste Text into your GitHub post.

Trust me.  It works.

**Comment by Mattmjs** [comment]
I tried with the stable extension and it failed with the error. I
tried in *Version
8.2.2.8a1 (beta) *and it worked.
I downloaded 10% of it before quitting. If you haven't already, try the
beta version.

**Reply by GrampaWildWilly** [reply]
I haven't checked lately but a few weeks ago, the link to the VDH beta in the Wiki was faulty.  It was not the latest beta.  What you have to do is:

1. Get the backlevel beta from the Wiki.
2. Go into the Firefox settings for add-ons (about:addons in the URL bar or Ctrl+Shift+a).
3. Click the 3 dots in the entry for VDH.  This opens a menu.
4. Click Manage on that menu.
5. Disable automatic updates.  (Change the radio button for automatic updates from On to Off.)  That will give you a Check for Updates button.
6. Click the Check for Updates button.
7. That will offer you an update.  It will be the latest beta.
8. Install it.
9. Retest.

Every few days, click the Check for Updates button to see if yet a newer beta has been released.  8.2.2.8a1 beta has been the latest for a few weeks now so you probably don't need to be checking every few hours.  Every couple of days should be good enough.  Or wait until somebody posts here about a new version.

**Reply by jeff9060** [reply]
I already test beta version.sometime can download.if choice 3 or more file will be error.
